ext/curl/multi.c

```diff
@@ -47,25 +48,34 @@
 
 #define SAVE_CURLM_ERROR(__handle, __err) (__handle)->err.no = (int) __err;
 
-/* {{{ proto resource curl_multi_init(void)
+/* CurlMulti class */
```

Comment:
```suggestion
/* CurlMultiHandle class */
```

Reply:
I've fixed these in the commit against master.